ext_conf_template.txt

```diff
@@ -0,0 +1,5 @@
+# cat=basic//1; type=string; label= Comma separated list of fields to set recursively : Configure what columns are going to be displayed in the BE module.
+fields = no_index, no_follow
```

Comment:
I suggest to use the "old default" `no_search` as default and use no_index/no_follow only in the example.